multiraft/multiraft.go

```diff
@@ -903,13 +909,17 @@ func (s *state) sendMessage(groupID uint64, msg raftpb.Message) {
 	snapStatus := raft.SnapshotFinish
 	if err != nil {
 		log.Warningf("node %v failed to send message to %v: %s", s.nodeID, nodeID, err)
-		s.multiNode.ReportUnreachable(msg.To, groupID)
+		if groupID != noGroup {
```

Comment:
If we fail to send a heartbeat we should probably fan that out into a ReportUnreachable for every group on the node.